lib/info.js

```diff
@@ -127,10 +129,13 @@ function gotConfig(id, options, additional, config, appendConfig, callback) {
   } catch (err) {
     return callback(new Error('Error parsing config: ' + err.message));
   }
+  var path = INFO_PATH;
+  var lang = '?hl' + (options.lang || 'en');
```

Comment:
`hl` is already part of the url below in the query (line 144). You can put this there, no parens required.